### PR TITLE
Updated login page

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -3,67 +3,46 @@
 @section('bodyClass', 'login')
 
 @section('content')
-    <div class="container">
-        <div class="row">
-            <div class="form-bg">
-                <div class="logo">
-                    <svg viewBox="0 0 1150 296" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
```
Comment (jbrooksuk) [inline resources/views/auth/login.blade.php]: I'm glad this has gone!

**Comment by jbrooksuk** [comment]
Merging this so all the images are local. I'm not too fussed. If a company want to remove their logo, they can ask and it'll be gone in a minor patch version.

We'll update this image anyway.

**Comment by GrahamCampbell** [comment]
We have to include every companies license in here by law.

**Comment by jbrooksuk** [comment]
Can we not do that in the LICENSE file?

**Comment by GrahamCampbell** [comment]
No. Some we actually are not allowed to put in here anyway if they have trademarks.

**Pull request merged**